pom.xml

```diff
@@ -117,8 +117,10 @@
         <kxml2.servicemix.version>2.3.0_3</kxml2.servicemix.version>
         <!-- double-check downstream projects before changing jackson version -->
         <fasterxml.jackson.version>2.7.5</fasterxml.jackson.version>
-        <cxf.version>3.1.4</cxf.version>
-        <httpclient.version>4.4.1</httpclient.version>
+        <cxf.version>3.1.10</cxf.version>
+        <httpcomponents.httpclient.version>4.5.2</httpcomponents.httpclient.version>
+        <httpcomponents.httpcore.version>4.4.4</httpcomponents.httpcore.version>
+        <httpclient.version>4.4.1</httpclient.version> <!-- kept for compatibility in 0.11.0-SNAPSHOT, remove after -->
```

Comment:
Bumped to `4.5.2` in #570. It could break some projects due to the version mismatch between core and client, but that's better than having a different version from Brooklyn's.